models/articles.models.js

```diff
@@ -30,4 +31,21 @@ exports.incrementArticleVotes = (id, body) => {
             }
         })
     }
-}
\ No newline at end of file
+}
+
+exports.getTheseComments = (id) => {
+    const comments = db.query('SELECT * FROM comments WHERE article_id = $1', [id])
+    
+    const thisArticleExists = valueExists('articles', 'article_id', id)
+
+    return Promise.all([comments, thisArticleExists])
+    .then(([comments, thisArticleExists]) => {
+        console.log(comments.rows)
```

Comment:
Please remove console logs. If you are asking for a pull request, you are saying that this code is ready to push to main and should not need troubleshooting/debugging